release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2613,8 +2613,8 @@ link:https://access.redhat.com/solutions/6750371[{product-title} 4.9.22 containe
 
 * Before this update, if you repeatedly clicked links to get details for resources such as custom resource definitions or pods and the application encountered multiple code reference errors, it crashed and displayed a `t is not a function` error. This update resolves the issue. Now, when an error occurs, the application resolves a code reference and stores the resolution state so that it can correctly handle additional errors. The application no longer crashes when code reference errors occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2022158[BZ#*2022158*])
 
-[id="ocp-4-9-22-updating"]
-==== Updating
+[id="ocp-4-9-22-upgrading"]
+==== Upgrading
```

Comment:
See previous. Check throughout. I'll not mark anymore of these. I see "update" in line 2640.